src/services/metrics-recorder.ts

```diff
@@ -156,7 +159,7 @@ export class MetricsRecorder {
           elapsedTime,
           result,
           timeout,
-          sessionKey
+          pocketSession
```

Comment:
This needs to be a string -- the hashed sessionKey -- here it is the full session going to the logs.

Reply:
on this case inside the function the variable is destructure and uses the `sessionKey` and `sessionNodes` whenever is necessary, otherwise would have to pass both variables which would achieve the same thing